app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,36 @@
+!!!
+%html
+  %head
+    %meta{:charset => "utf-8"}/
+    %title Instagram
+    %link{:href => "https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css", :rel => "stylesheet"}
+    %script{:src => "https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"}
+    %link{:href => "https://fonts.googleapis.com/css?family=Open Sans", :rel => "stylesheet", :type => "text/css"}
+    = stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track' => true
+    = javascript_include_tag 'application', 'data-turbolinks-track' => true
+    = csrf_meta_tags
+  %body
+    .navbar.navbar-inverse.navbar-static-top{:role => "navigation"}
+      .container
+        .navbar-header
+          %button.navbar-toggle{"data-target" => ".navbar-collapse", "data-toggle" => "collapse", :type => "button"}
+            %span.sr-only Toggle navigation
+            %span.icon-bar
+            %span.icon-bar
+            %span.icon-bar
+          %a.navbar-brand{:href => "/", :style => "font-family: Open Sans; font-size: 30px;"} Instagram
```

Comment:
Line is too long. [103/80]<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace